src/commands/test/cheatcodes/curry_run_from_entrypoint.py

```diff
@@ -0,0 +1,13 @@
+def curry_run_from_entrypoint(fn, test_runner):
```

Comment:
I think name is a bit misleading because it doesn't only curry a function but edits input data either.

Reply:
I'll change it to `inject_protostar_hint_locals`.